projects/homework_102_nourish/src/main.c

```diff
@@ -0,0 +1,44 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include "interrupt.h"   // interrupts are required for soft timers
```

Comment:
Nitpick: there should be a blank line between <standard library> headers and MS-specific headers.